ADA/databricks_notebooks.qmd

```diff
@@ -0,0 +1,54 @@
+
+You can create a notebook in your workspace, either in a folder or a repository.\
+To do this locate the folder/repository you want to create the notebook in then click the 'Create' button and select Notebook.
+
+::: callout-tip
+It's generally recommended that any notebooks used for core business processes are created in a repository linked to GitHub/DevOps where they can be version controlled.
+:::
+
+Once you've created a notebook it will automatically be opened. Any changes you made are saved in real time. In order to 'save' a snapshot of your work it is recommended to use git commits.
+
+You can change the title from 'Untitled Notebook *\<timestamp\>*', and set it's default language in the drop down immediately to the right of the notebook title.
+
+![](/images/ada-notebook.png)
+
+The default language is the language the notebook will assume all code chunks are written in. In the screenshot above the default language is 'R', so all chunks will be assumed to be written in R unless otherwise specified.
+
+You can also add markdown cells to add text, links and graphics to your notebook in order to document the processing done within it.
+
+To add a new code of markdown chunk move the mouse above or below another chunk and the buttons '+Code' and '+Text' will appear.
+
+![](/images/ada-notebook-add-chunk.png)
+
+To run code chunks you'll first need to attach your compute resource to it by clicking the 'Connect' button in the top right hand side of the page.
+
+![](/images/ada-notebook-attach.png)
+
+You can run a code chunk either by pressing the play button in the top left corner of the chunk, or by pressing Ctrl + Return/Enter on the keyboard. Any outputs that result from the code will be displayed underneath the chunk.
+
+![](/images/ada-notebook-chunk-output.png)
+
+Everything ran in a notebook is in it's own 'session' meaning that later chunks have access to variables, functions, etc. that were defined above. Chunks can be ran manually, however doing this runs the risk of running code out of order and may consequently produce unexpected results. To avoid this all chunks can be ran in order from the beginning of the Notebook using the 'Run all' button at the top of the page, alternatively you can 'Run all above' or 'Run all below' from any code chunk.
+
+Notebooks cannot share a session with another Notebook so bear this in mind when constructing your workflows. If you need to pass data between notebooks it can be written out to a table in the unity catalog using SQL / R / Python / Scala as you would write to a SQL table in SQL Server. This can then be accessed from later notebooks.
```

Comment:
I am assuming there aren't any access / permission issues that'd cause people problems with writing to the unity catalog? Like this is just a thing we can all do?

Reply:
If your account has access to the catalog you're trying to write to you can write to it, just the same as you can write to SQL Server now.